<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyFilesClass</class>
 <widget class="QMainWindow" name="CopyFilesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>738</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>738</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量操作文件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>271</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加源文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_retainWorkSpace">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>拷贝时保留原工作路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_srcWorkSpacePath">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>95</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_selectSrcWorkSpace">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>118</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:blue;
background:transparent;</string>
    </property>
    <property name="text">
     <string>指定源文件工作路径</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_forceCopy">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>若目标路径文件存在时始终</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标文件夹</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>275</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_apply">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>195</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>默认链接方式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_createWinLink">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>快捷方式（仅windows）</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_createSymLink">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>软链接</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_createHardLink">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>硬链接</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_forceCopy" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radio_forceCopy">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>替换</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_forceNoCopy">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>不替换</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>(若不勾选时，遇到重名问题，将提示用户选择。)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>button_add</tabstop>
  <tabstop>button_delete</tabstop>
  <tabstop>check_retainWorkSpace</tabstop>
  <tabstop>lineEdit_srcWorkSpacePath</tabstop>
  <tabstop>button_selectSrcWorkSpace</tabstop>
  <tabstop>check_forceCopy</tabstop>
  <tabstop>radio_forceCopy</tabstop>
  <tabstop>radio_forceNoCopy</tabstop>
  <tabstop>radio_createWinLink</tabstop>
  <tabstop>radio_createSymLink</tabstop>
  <tabstop>radio_createHardLink</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>button_apply</tabstop>
 </tabstops>
 <resources>
  <include location="CopyFiles.qrc"/>
 </resources>
 <connections/>
</ui>
